### PR TITLE
Update changelog for 0.13.0

### PR DESCRIPTION
**Pull request by yuryybk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+### Technical improvements
+
+- Integrated new rendering engine for Detection, Segmentation, Lane visualizations & AR
+- Recognizing Okinawa as part of Japan
+
+### User experience
+
+- Added Replay & Recording modes 
+- Added support of the reverse landscape orientation
```
Comment (kiryldz) [inline CHANGELOG.md:13]:
```suggestion
- Added reverse landscape orientation support
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.13.0
+
+### Technical improvements
+
+- Integrated new rendering engine for Detection, Segmentation, Lane visualizations & AR
+- Recognizing Okinawa as part of Japan
```
Comment (kiryldz) [inline CHANGELOG.md:8]: I'd use common verb style here. Smth like
```suggestion
- Added Okinawa as part of Japan
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+### User experience
+
+- Added Replay & Recording modes 
+- Added support of the reverse landscape orientation 
+- Added Lane Detection mode 
+- Added Camera default resolution: 960x540
```
Comment (kiryldz) [inline CHANGELOG.md:15]:
```suggestion
- Changed camera default resolution to 960x540 if supported
```

Comment (yuryybk) [reply CHANGELOG.md]: Changed

File: CHANGELOG.md (hunk 2 above)
Comment (yuryybk) [reply CHANGELOG.md:8]: Changed

File: CHANGELOG.md (hunk 1 above)
Comment (yuryybk) [reply CHANGELOG.md:13]: Changed

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+### Technical improvements
+
+- Integrated new rendering engine for Detection, Segmentation, Lane visualizations & AR
+- Added Okinawa as part of Japan
+
+### User experience
+
+- Added Replay & Recording modes 
+- Added reverse landscape orientation support
+- Added Lane Detection mode
```
Comment (kiryldz) [inline CHANGELOG.md:14]: To align with phrase above.
```suggestion
- Added Lane Visualization mode 
```

Comment (yunikkk) [reply CHANGELOG.md:14]: Maybe `Added lane visualization to Lane Detection mode` ?

Comment (kiryldz) [reply CHANGELOG.md:14]: We have then `Detection` and `Lane Detection` modes.
Not sure about it.

Comment (yuryybk) [reply CHANGELOG.md:14]: `Added lane visualization to Lane Detection mode` looks ok.
Btw we really have both Object Detection and Lane Detection modes

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.13.0
+
+### Technical improvements
+
+- Integrated new rendering engine for Detection, Segmentation, Lane visualizations & AR
```
Comment (kiryldz) [inline CHANGELOG.md:7]: Well, I guess there should be `Lane Detection` then.
However I'm already a bit confused so feel free to leave as is.

Comment (yuryybk) [reply CHANGELOG.md:7]: @kiryldz How about `Integrated new rendering engine for Object & Lane Detection, Segmentation, AR`?

Comment (kiryldz) [reply CHANGELOG.md:7]: I like it!

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.13.0
+
+### Technical improvements
+
+- Integrated new rendering engine for Object & Lane Detection, Segmentation & AR
```
Comment (kiryldz) [inline CHANGELOG.md:7]: I guess here and everywhere below we should have `Integrate` / `Add` etc

Comment (yuryybk) [reply CHANGELOG.md]: yeah, fixed

**Pull request merged**